autospec/build.py

```diff
@@ -360,6 +360,7 @@ def package(filemanager, mockconfig, mockopts, cleanup=False):
         "--enable-plugin=ccache",
         f"--uniqueext={uniqueext}",
         cleanup_flag,
+        mockopts,
```

Comment:
Should we have two different opts for srpm and brpm eventually? Well I'll wait for that until we have a use case.

Reply:
Potentially, yes. I also would wait until we have a use case.